battery_parameter_test.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+import Battery_Parameter_Handler
+
+class test_battery_current_ranges(unittest.TestCase):
+    
+  def test_battery_range(self):    
+    self.assertTrue(Battery_Parameter_Handler.process_information(["Charge_rate" ,"Temperature"],"Console"))    
+    
+  #def test_alertType(self):
```

Comment:
Why comment all the good tests? You could change the `alertType` to something like `"TEST"`, which would simply test the parameters, format, etc.